qt/Application.cc

```diff
@@ -81,13 +81,13 @@ bool loadTranslation(QTranslator& translator, QString const& name, QLocale const
 [[nodiscard]] auto makeWindowIcon()
 {
     // first, try to load it from the system theme
-    if (auto icon = QIcon::fromTheme(QStringLiteral("transmission")); !icon.isNull())
+    if (auto icon = QIcon::fromTheme(QStringLiteral(":/icons/transmission-tray.ico")); !icon.isNull())
     {
         return icon;
     }
 
     // if that fails, use our own as the fallback
-    return QIcon{ QStringLiteral(":/icons/transmission.svg") };
+    return QIcon{ QStringLiteral(":/icons/transmission-tray.svg") };
```

Comment:
As the icon is always there in the resources, I suppose it'll always be found, so this fallback here seems unnecessary, as well as having icon in two formats (ico and svg). To be fair, the fallback was probably not needed before your changes either, so not expecting you to change anything here, unless you want to.

Reply:
That fallback is because of my unsure footing between how Windows and Linux might handle the image formats in CI and the build process.